Integrations/integration-CounterTack.yml

```diff
@@ -1765,14 +1966,203 @@ script:
         - name: file_id
           required: true
           description: The ID of the extracted file
-      description: Download an extracted file
-    - name: countertack-delete-extracted-file
+      outputs:
+        - contextPath: File.Size
+          description: The size of the extracted file (bytes).
+          type: number
+        - contextPath: File.SHA1
+          description: The SHA1 digest of the file contents.
+          type: string
+        - contextPath: File.SHA256
+          description: The SHA256 digest of the file contents.
+          type: string
+        - contextPath: File.Name
+          description: The file name
+          type: string
+        - contextPath: File.SSDeep
+          description: The SSDEEP digest of the file contents.
+          type: string
+        - contextPath: File.EntryID
+          description: EntryID of the file
+          type: string
+        - contextPath: File.Info
+          description: File information
+          type: string
+        - contextPath: File.Type
+          description: File type
+          type: string
+        - contextPath: File.MD5
+          description: The MD5 digest of the file contents.
+          type: string
+        - contextPath: File.Extension
+          description: The extension of the file (zip)
+          type: string
+      description: Download an extracted file in ZIP format. The password to unlock
+        the zip file is sentinel
+    - name: countertack-search-events
       arguments:
-        - name: file_id
+        - name: expression
           required: true
-          description: The ID of the extracted file
-      description: 'Delete an extracted file from the endpoint.'
+          description: The CQL expression to be used for the search (e.g events.event_type=basic)
+      outputs:
+        - contextPath: CounterTack.Event.Results.SourceProcessTimeStarted
```

Comment:
not sure why the `Results` level is needed here
please remove it, so for example this will be `CounterTack.Event.SourceProcessTimeStarted`